Core/Core/Discussions/DiscussionDetailsViewController.swift

```diff
@@ -232,9 +232,9 @@ public class DiscussionDetailsViewController: UIViewController, ColoredNavViewPr
                 dueSection.isHidden = false
             }
 
-            if topic.first?.groupTopicChildren == nil, submissionsSection.isHidden {
+            if topic.first?.groupTopicChildren == nil, submissionsSection.isHidden, let courseID = courseID {
                 let controller = SubmissionBreakdownViewController.create(
-                    context: context,
+                    courseID: courseID,
```

Comment:
made this always require a courseID to make sure we don't try to use the group context.